generators/app/templates/src/app/_app.component.ts

```diff
@@ -8,7 +8,7 @@ import { Component, OnInit } from '@angular/core';
 import { Router, NavigationEnd, ActivatedRoute } from '@angular/router';
 import { Title } from '@angular/platform-browser';
 import { TranslateService } from '@ngx-translate/core';
-import { merge } from 'rxjs';
+import { merge } from 'rxjs/observable/merge';
```

Comment:
No, starting from rxjs 6 this can (and must) safely be imported directly from `rxjs`

Reply:
I don't know why I thought it was the other way round 😅